x/ibc/06-solomachine/types/client_state.go

```diff
@@ -436,3 +415,53 @@ func setClientState(store sdk.KVStore, clientState clientexported.ClientState) {
 	bz := SubModuleCdc.MustMarshalBinaryBare(clientState)
 	store.Set(host.KeyClientState(), bz)
 }
+
+// validateVerificationArgs perfoms the basic checks on the arguments that are
+// shared between the verification functions.
+func validateVerificationArgs(
+	cs ClientState,
+	height uint64,
+	prefix commitmentexported.Prefix,
+	proof commitmentexported.Proof,
+	consensusState clientexported.ConsensusState,
+) error {
+	if cs.GetLatestHeight() < height {
+		return sdkerrors.Wrapf(
+			sdkerrors.ErrInvalidHeight,
+			"client state (%s) height < proof height (%d < %d)", cs.ID, cs.GetLatestHeight(), height,
```

Comment:
```suggestion
			"client state (%s) sequence < proof sequence (%d < %d)", cs.ID, cs.GetLatestHeight(), sequence,
```

using `sequence` instead of `height` I think makes more logical sense for solo machine. I'm planning on using that var for building sign bytes

Reply:
I'll be changing this downstream